### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by pbchase**
This adds an implementation guide and cleans up the developer notes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,15 +39,52 @@ The module is configurable at the system level to allow the subject line and bod
 
         10, 12, 20
 
+
+
+## How to Implement WUPS and Account Suspensions
+
+Implementing WUPS and/or activating REDCap account suspensions can require some careful planning to avoid annoying your users who have not logged in recently.  If you have never used account suspension on your REDCap host, activating it will cause all accounts that have not logged in within the _Period of inactivity_ to be suspended within 24 hours. If those people want their accounts reenabled they will have to ask the REDCap admin to reenable them.  That generates the kind of help desk workload WUPS was designed to _prevent_.
+
+To avoid the chaos of hundreds of accounts getting prematurely suspended, you can run a few SQL queries to adjust the last login dates and last activity dates for your REDCap users.  Done correctly, you can use WUPS to warn these users of the pending suspension, allow interested REDCap users to renew their account, and let the rest suspended normally.
```
Comment (marlycormar) [inline README.md]: > (...) and let the rest ~~suspended~~ suspend normally.

File: README.md [hunk 2]
```diff
@@ -34,15 +39,52 @@ The module is configurable at the system level to allow the subject line and bod
...
+
+The first step is to configure WUPS' _Days Before Suspension_.  In this example, we'll use `30, 15, 7, 3, 1`, but only the highest number affects our work. We've also set _Period of inactivity_ to 180 days. We want everyone who is approaching their date of suspension to receive every warning WUPS is configured to provide. To achieve that, _no one_ is allowed to be within 30 days of suspension when WUPS is turned on. This requires some accounts have their date of last login and last activity changed.
```
Comment (marlycormar) [inline README.md]: > This requires some accounts +**to**+ have their date of last login and last activity changed.

File: README.md [hunk 3]
```diff
@@ -34,15 +39,52 @@ The module is configurable at the system level to allow the subject line and bod
...
+
+To change the last login and last activity dates, we first need to identify who needs the change.  This query will return all the usernames of accounts will expire within the next 30 days when _Period of inactivity_ is set to 180 days:
```
Comment (marlycormar) [inline README.md]: > the usernames of accounts +**that**+ will expire within the (...)

File: README.md [hunk 4]
```diff
@@ -34,15 +39,52 @@ The module is configurable at the system level to allow the subject line and bod
...
+              when user_lastlogin is not null then user_lastlogin
+              when user_creation is not null then user_creation
+              end) as user_last_date
+         from redcap_user_information
+         where user_suspended_time is null
+         ) as my_user_info
+    where DATEDIFF(NOW(), user_last_date) > (180 - 30)
+    );
+
+With that temporary table created, it is a simple matter to change `user_lastactivity` and `user_lastlogin` to  random date between 120 and 150 days.
```
Comment (marlycormar) [inline README.md]: > (...) to +**a**+  random date between 120 and 150 days (...)

File: README.md [hunk 5]
```diff
@@ -34,15 +39,52 @@ The module is configurable at the system level to allow the subject line and bod
...
 
     update redcap_config set value="all" where field_name = "suspend_users_inactive_type";
     update redcap_config set value="1" where field_name = "suspend_users_inactive_send_email";
     update redcap_config set value="30" where field_name = "suspend_users_inactive_days";
 
-As this tool sends email, make sure the from address are configured correctly in your redcap system. This tool uses "Email Address of REDCap Administrator" in the REDCap Control Center, General Configuration tab and it's always a good idea to set a valid "universal 'FROM' email address".  Those can be set quickly via SQL if you are so inclined. Here's an example of how a lazy developer at the University of Florida might do that:
+As this tool sends email, make sure the from address is configured correctly in your REDCap system. This tool uses "Email Address of REDCap Administrator" in the REDCap Control Center, General Configuration tab.  It's also good idea to set a valid "universal 'FROM' email address" on that same page.  Those can be set quickly via SQL if you are so inclined. Here's an example of how a lazy developer at the University of Florida might do that:
```
Comment (marlycormar) [inline README.md]: > It's also +**a**+ good idea to set a valid "universal 'FROM' email address" on that same page (...)

**Comment by pbchase** [comment]
@marlycormar, commit 968ec67 addresses your feedback.

**Pull request merged**